roles/sfia/software_engineer_2.md

```diff
@@ -12,11 +12,11 @@ Contributor to digital, data and technology outcomes that improve society. They
 
 ### Autonomy
 
-Collaboratively leads planning of work, development of features and coordinating with stakeholders though continues to seek feedback and support on complex change.
+Collaboratively leads planning of work, development of features and coordinating with stakeholders through continues to seek feedback and support on complex change.
```

Comment:
"though" was correct in the current context

Reply:
I read this as though continuous feedback 😅